bin/upload.js

```diff
@@ -2,7 +2,10 @@
 
 var upload = require('..');
 var fs = require('fs');
-var argv = require('minimist')(process.argv.slice(2));
+var argv = require('minimist')(process.argv.slice(2), {
+    boolean: 'patch',
+    string: 'name'
```

Comment:
This addition makes this check for `name === true` never happen, since `name` will now equal an empty string `''` if it's used but no value is given. Feel free to remove this or update the boolean check: 

* if the name property exists (this is tricky since if the value is an empty string, `argv.name` will be false)
* and if the name property has a length of zero